UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/Interfaces/INMDataService.cs

```diff
@@ -10,7 +10,8 @@ public interface INMDataService
         Task<ShowWeeklyFilesResponseModel> GetWeeklyFilesResponseModelsAsync(int year, int week, string correlationId);
         Task<byte[]> DownloadFssFileAsync(string batchId, string fileName, string correlationId, string frequency);
         Task<ShowNMFilesResponseModel> GetLeisureFilesAsync(string correlationId);
-        Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, string correlationId);
+        Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, bool isDistributorUser, string correlationId);
```

Comment:
No need to pass isDistributorUser parameter from input.
```suggestion
        Task<byte[]> DownloadFSSZipFileAsync(string batchId, string fileName, string correlationId);
```

Reply:
Done